api/pom.xml

```diff
@@ -61,6 +61,11 @@
             <groupId>commons-lang</groupId>
             <artifactId>commons-lang</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.ehrbase.openehr</groupId>
+            <artifactId>terminology</artifactId>
+            <version>${project.version}</version>
```

Comment:
The current pattern is to add the version in the dependence management of the main.pom such that in the module.poms not version is needed.

Reply:
fixed with the new commit